### ISSUE TITLE
Add a note to README that the plugin won't ask for an API key if you've installed other wakatime plugins

### ISSUE THREAD

**Issue by pixelbandito** [wakatime/jetbrains-wakatime#121]
Looks like the prompt for entering your key (in the instructions and animation) doesn't happen if you already have a `.wakatime.cfg` file in your home directory. (If you've added plugins for other editors, for instance.)

Adding a note to this effect to the README file might save ppl some time and confusion.

**Comment by gandarez** [comment]
@pixelbandito  you're right. Api Key will be asked only if it's your first time using WakaTime plugin.

**Comment by alanhamlett** [comment]
The website install instructions contain a note about this in the question mark circle icon tooltip:

https://wakatime.com/intellij-idea
